Acumatica REST API Console Application/RESTExample.cs

```diff
@@ -17,28 +21,61 @@ public static void ExampleMethod(string siteURL, string username, string passwor
-				}
+                Console.WriteLine("Reading Accounts...");
+                var accountApi = new AccountApi(configuration);
+                var accounts = accountApi.GetList(top: 5);
+                foreach (var account in accounts)
+                {
+                    Console.WriteLine("Account Nbr: " + account.AccountCD.Value + ";");
+                }
 
-				Console.WriteLine("Reading Sales Order by Keys...");
-				var salesOrderApi = new SalesOrderApi(configuration);
-				var order = salesOrderApi.GetByKeys(new List<string>() { "SO", "SO005207" });
-				Console.WriteLine("Order Total: " + order.OrderTotal.Value);
+                Console.WriteLine("Reading Sales Order by Keys...");
+                var salesOrderApi = new SalesOrderApi(configuration);
+                var order = salesOrderApi.GetByKeys(new List<string>() { "SO", "SO005207" });
+                Console.WriteLine("Order Total: " + order.OrderTotal.Value);
 
 
-				var shipmentApi = new ShipmentApi(configuration);
-				var shipment = shipmentApi.GetByKeys(new List<string>() { "002805" });
-				Console.WriteLine("ConfirmShipment");
-				shipmentApi.WaitActionCompletion(shipmentApi.InvokeAction(new ConfirmShipment(shipment)));
+                var shipmentApi = new ShipmentApi(configuration);
+                var shipment = shipmentApi.GetByKeys(new List<string>() { "002805" });
+                Console.WriteLine("ConfirmShipment");
+                shipmentApi.WaitActionCompletion(shipmentApi.InvokeAction(new ConfirmShipment(shipment)));
 
-				Console.WriteLine("CorrectShipment");
-				shipmentApi.WaitActionCompletion(shipmentApi.InvokeAction(new CorrectShipment(shipment)));
-			}
+                Console.WriteLine("CorrectShipment");
+                shipmentApi.WaitActionCompletion(shipmentApi.InvokeAction(new CorrectShipment(shipment)));
+
+                Console.WriteLine("File Upload/Download");
+                order = salesOrderApi.GetByKeys(new List<string>() { "SO", "SO005207"}, expand: "files");
+
+                byte[] initialData = Encoding.UTF8.GetBytes("Acumatica is awesome");
+                string fileName = "TestFile.txt";
+                salesOrderApi.PutFile("SO/SO005207", fileName, initialData);
+                
+                order = salesOrderApi.GetByKeys(new List<string>() { "SO", "SO005207" }, expand: "files");
+
+                if (order.Files.Any(fl => fl.Filename.EndsWith(@"\" + fileName))){
+                    Console.WriteLine($"The file {fileName} was uploaded successfully");
+                }
+
+                Uri basePath = new Uri(siteURL);
+
+                foreach (FileLink fileLink in order.Files)
+                {
+                    var test = configuration.BasePath + fileLink.Href;
+                    var content = authApi.Configuration.ApiClient.RestClient.DownloadData(new RestRequest(basePath.Scheme+ "://" + basePath.Authority + fileLink.Href));

```

Comment:
Example code is not supposed to utilize internal logic of the Client. Ideally (it is not the case right now), the example code should not even know that we use RestSharp inside our client.